src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BOUND = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(MAX_BOUND));
+        ball.setColor(colorSupplier.getRandomColor());
+        return ball;
```

Comment:
it's okay as is, but if you had a constructor for ball (instead of using setters) you would be able to return directly (return new Bal(..) and this could be one-liner